test/mavsdk_tests/mavsdk_test_runner.py

```diff
@@ -359,6 +369,15 @@ def run_test_case(self, test: Dict[str, Any],
                 print("  - {}".format(runner.get_log_filename()))
         return is_success
 
+    def clear_params(self) -> None:
+        param_files = glob.glob(
+            "build/px4_sitl_default/tmp/rootfs/eeprom/parameters_*",
```

Comment:
I prefer if this runs under a different rootfs directory, like `build/px4_sitl_default/tests`, then you can just remove the whole directory on startup, and thus a more predictable environment.

Reply:
Good idea!